src/helpers/assets.ts

```diff
@@ -1,8 +1,18 @@
 import lang from 'i18n-js';
-import { chunk, compact, groupBy, isEmpty, slice, sortBy } from 'lodash';
-import { add, convertAmountToNativeDisplay, greaterThan } from './utilities';
+import chunk from 'lodash/chunk';
+import compact from 'lodash/compact';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
Is the eventual plan to go back and replace these again too? https://github.com/rainbow-me/rainbow/pull/3509/files#diff-dfbbddbff81e448260b519c9739b639b428c4ba3bb3a4aab4e6e82eab651c367R17